spread/client.go

```diff
@@ -79,14 +79,29 @@ func (c *Client) dialOnReboot() error {
 
 	waitConfig := *c.config
 	waitConfig.Timeout = 5 * time.Second
+	uptimeDelta := 3 * time.Second
```

Comment:
Let's please make the use case for this variable more clear by naming it _uptimeChanged_.